.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 *.pyc
 *.iml
+logging/
+config_dev.yaml
+passfile.txt
```

Comment:
My point is that cryptography is in general safer than a shared secret and thus should be preferred.   Plus, it may be what the user has on hand or is comfortable. Personally, I think we should support both. I suppose we can start with password and add this to the next iteration.